### ISSUE TITLE
Uninstaller fails to remove audio devices from output list

### ISSUE THREAD

**Issue by dpaustin** [ExistentialAudio/BlackHole#432]
On Big Sur 11.4, the uninstaller doesn't remove the audio devices from the audio output list. After running the uninstaller package, I had to also manually disconnect any audio devices from the BlackHole devices within "Audio MIDI Setup" and then restart to finally remove BlackHole devices from the device list. At least you might add these instructions to the manual uninstall instructions.

**Comment by devinroth** [comment]
Not sure exactly what you mean by “disconnect any audio devices from the BlackHole devices”.

Devin Roth

**Comment by dpaustin** [comment]
As shown below, the Blackhole devices still display in the "Audio MIDI Setup" after uninstalling. I had to uncheck the “Use” box to stop displaying them in the device list main menu.




At least they don’t show in the device list main menu anymore:


I’m still figuring out how to remove the device names from the MIDI dialoging the first screen shot.

Thx.

**Comment by dpaustin** [comment]
I manually deleted the audio devices from MIDI dialog using the “-“ button at the list bottom.

**Comment by devinroth** [comment]
Screenshots? I don’t see any screenshots.

Devin Roth

**Comment by dpaustin** [comment]
Here's the dialog where blackhole still appears after uninstall. As above, I had to also manually disconnect any audio devices from the BlackHole devices within "Audio MIDI Setup". After restarting once more, the audio devices finally disappeared from audio outputs, and I manually deleted the audio devices from MIDI dialog using the “-“ button at the list bottom.
<img width="650" alt="PastedGraphic-1" src="https://user-images.githubusercontent.com/16466416/127756413-c08c0be9-fa6e-45c1-9653-4974f0d5314f.png">
